R/semi.sup.R

```diff
@@ -237,16 +237,28 @@ semi.sup <- function(
         snow::clusterEvalQ(cl, library(recetox.aplcms))
         
         cpu_time <- system.time(
-          aligned <-
-            feature.align(
-              f2,
-              min_occurrence = min.exp,
-              mz_tol_relative = align.mz.tol,
-              rt_tol_relative = align.rt.tol,
-              mz_max_diff = 10 * mz.tol,
-              mz_tol_absolute = max.align.mz.diff,
-              sample_names = sample_names
-            )
+            aligned <- {
+                res <- compute_clusters(
+                    f2,
+                    align.mz.tol,
+                    align.rt.tol,
+                    10 * mz_tol,
+                    max.align.mz.diff,
+                    FALSE,
+                    sample_names
+                )
+                
+                aligned <- create_aligned_feature_table(
+                    dplyr::bind_rows(res$feature_tables),
```

Comment:
dplyr can be added as an import for the whole script to keep things consistent

Reply:
Good catch. It's actually just one function that's used twice throughout the script, so importing just the `bind_rows` function for the whole script would be even cleaner.

Reply:
I imported just `bind_rows` in [d6fd115](https://github.com/RECETOX/recetox-aplcms/pull/154/commits/d6fd115253dc545ebd655fdc712a002730fdf18f).